api_scrape.py

```diff
@@ -1,17 +1,31 @@
-from bs4 import BeautifulSoup
-import urllib.request
-
+import requests
+from requests.exceptions import HTTPError
 
 urls = open('appIDs.txt', 'r')
 
-with urls as f:
-    url = f.readline()
-    req = urllib.request.Request(url)
-    resp = urllib.request.urlopen(req)
-    respData = resp.read()
-    print(respData)
 
+try:
+    with urls as f:
+        url = f.readline()
+        response = requests.get(url)
+        response.raise_for_status()
+        json_response = response.json()
+        print(json_response)
+except HTTPError as http_err:
+    print(f'An HTTP error has occurred: {http_err}')
+except Exception as err:
+    print(f'Another error has occurred: {err}')
+
+
+for key, value in json_response.items():
+    categories = json_response[key]['data']['categories']
+    for k in categories:
```

Comment:
Variable naming suggestion: I believe `k` is one of the categories, right? Could we name it `category` instead?

Reply:
It isn't labeled a category oddly. It IS two more keys and values

Reply:
Oh, so `categories` is another dictionary? So `k` would be the category key?

Reply:
Yup! Categories is indeed another dictionary so was trying to keep a similar pattern to essentially say 'these are keys too!'